File: Beispiele/Java2Kotlin/src/main/kotlin/javakotlinworkshop/java/ClassDemo.java
package javakotlinworkshop.java;

public class ClassDemo {

    public static void main(String[] args) {
        Person person = new Person("John", 50);
        person.setName("Max");
        person.setAge(42);
        print(person);
    }

    private static void print(Person person) {
        System.out.println(String.format("%s, %d",
                person.getName(),
                person.getAge()));
    }
}

File: Beispiele/Java2Kotlin/src/main/kotlin/javakotlinworkshop/kotlin/ClassDemo.kt
package javakotlinworkshop.kotlin

fun main(args: Array<String>) {
    val person = Person("John", 50)
    person.name = "Max"
    person.age = 42
    print(person)
}

fun print(person: Person) {
    println("${person.name}, ${person.age}")
}

class Person(var name: String, var age: Int)


